config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -39,8 +39,10 @@ periodics:
               rm -rf /usr/local/go
               tar -C /usr/local -xzf go.tgz
               make test KUBE_RACE=-race
-  - name: periodic-kubernetes-containerd-conformance-test-ppc64le
+  - name: test-periodic-kubernetes-containerd-conformance-test-ppc64le
```

Comment:
any specific reason for this `test-` prefix part of this PR?

Reply:
Hmm, it was added by mistake. Will remove it in the next patch.